jwst/extract_1d/extract.py

```diff
@@ -266,7 +275,11 @@ def get_extract_parameters(
         extract_params['ystop'] = shape[-2] - 1  # last pixel in Y
         extract_params['extract_width'] = None
         extract_params['src_coeff'] = None
-        extract_params['bkg_coeff'] = None
+        extract_params['bkg_coeff'] = None  # no background sub.
+        extract_params['smoothing_length'] = 0  # because no background sub.
+        extract_params['bkg_fit'] = None  # because no background sub.
+        extract_params['bkg_order'] = 0  # because no background sub.
+        extract_params['subtract_background'] = False
```

Comment:
It looks like this block is not unit tested.  Perhaps a small unit test that just checks that this function works as expected when called with `ref_dict=None`

Reply:
Agreed. Fortunately it is at least tested by 1 or 2 regression tests, specifically the WFSS mode, which does not use an extract1d ref file and hence the `ref_dict` is None.